techempower/scripts/perf/run-tfb-qrh-openshift.sh

```diff
@@ -24,7 +24,7 @@ SCRIPT_REPO=${PWD}
 
 CLUSTER_TYPE="openshift"
 pushd ".." > /dev/null
-HYPERFOIL_DIR="${PWD}/hyperfoil-0.13/bin"
```

Comment:
Suggest to have the versions defined in one place and the rest of the places to just reference it. This should work across both files, so maybe it should be defined in the common file

Reply:
Did the required changes.